packages/ui/src/utils/popover.ts

```diff
@@ -0,0 +1,29 @@
+import styled, { keyframes } from 'styled-components';
+
+const scaleIn = keyframes`
+  from {
+    opacity: 0;
+    transform: scale(0);
+  }
+  to {
+    opacity: 1;
+    transform: scale(1);
+  }
+`;
+
+export const PopoverContent = styled.div`
+  display: flex;
+  flex-direction: column;
+
+  width: 240px;
+  max-width: 320px;
+  padding: ${props => props.theme.spacing(3)} ${props => props.theme.spacing(2)};
+
+  background: ${props => props.theme.color.other.dialogBackground};
+  border: 1px solid ${props => props.theme.color.other.tonalStroke};
+  border-radius: ${props => props.theme.borderRadius.sm};
+  backdrop-filter: blur(${props => props.theme.blur.lg});
+
+  transform-origin: var(--radix-tooltip-content-transform-origin);
```

Comment:
Ah, just noticed this variable name. It should be `--radix-popover-content-transform-origin` ([docs](https://www.radix-ui.com/primitives/docs/components/popover#content)). I must have missed this when reviewing the popover PR.

Since Radix's dropdown menu also has a CSS variable (`--radix-dropdown-menu-content-transform-origin` — [docs](https://www.radix-ui.com/primitives/docs/components/dropdown-menu#content)), the CSS variable name should probably be passed in as a parameter.